src/redis/driver_impl.go

```diff
@@ -36,7 +36,10 @@ func poolTrace(ps *poolStats, healthCheckActiveConnection bool, srv server.Serve
 				ps.connectionTotal.Add(1)
 				ps.connectionActive.Add(1)
 				if healthCheckActiveConnection && srv != nil {
-					srv.HealthCheckOK()
+					err := srv.HealthChecker().Ok(server.RedisHealthComponentName)
+					if err != nil {
+						logger.Errorf("Unable to update health status: %s", err)
+					}
```

Comment:
This is new also, right? Seems like blocking on redis health should also be configurable as it's a change? Some for the other change that is not config guarded including sigterm? Sigterm seems OK to not config check, though it should be documented. But I think the redis change should be config guarded?

Reply:
With the "Config" health component config guarded, the behavior is the same as before. So I believe it should be fine to have the above not config guarded.
I can add more documentation about the redis and sigterm health components.

Reply:
Let me know what you think :)